### PR TITLE
- Initial commit

### PR DESCRIPTION
**Pull request by ianisms**

### CODE REVIEW

File: PostGreKeyRotation/Deployment/pg-keyrotation-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    - task: AzureCLI@2
+      displayName: 'Deploy Staging'
+      inputs:
+        failOnStandardError: true
+        azureSubscription: '<azure-subscription-id>'
+        scriptType: 'bash'
+        scriptLocation: 'scriptPath'
+        scriptPath: '$(System.DefaultWorkingDirectory)/PostGreKeyRotation/Deployment/scripts/pg-deploy-staging.sh'
+        workingDirectory: '$(System.DefaultWorkingDirectory)/PostGreKeyRotation/Deployment/scripts'
+        arguments: '"$(currentActiveRole)" "$(buildTag)"'
```
Comment (jwendl) [inline PostGreKeyRotation/Deployment/pg-keyrotation-pipeline.yml]: We had to add two more parameters here in the script: DBHOSTNAME and DBNAME

File: PostGreKeyRotation/Deployment/scripts/pg-deploy-staging.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+BUILDTAG=$2
+
+if [ -z "${ACTIVEROLE}" ]; then
+    echo "Usage:  $0 <active role>"
+    exit 1;
+else
+    echo "ACTIVEROLE: ${ACTIVEROLE}"
+    echo "BUILDTAG: ${BUILDTAG}"
+
+    DBHOSTNAME="<database-server-name>"
```
Comment (jwendl) [inline PostGreKeyRotation/Deployment/scripts/pg-deploy-staging.sh]: Let's move this to parameters.

**Review by jwendl — CHANGES_REQUESTED** [review]

File: PostGreKeyRotation/Deployment/scripts/pg-deploy-production.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+BUILDTAG=$2
+
+if [ -z "${ACTIVEROLE}" ]; then
+    echo "Usage:  $0 <active role> <build tag>"
+    exit 1;
+else
+    echo "ACTIVEROLE: ${ACTIVEROLE}"
+    echo "BUILDTAG: ${BUILDTAG}"
+
+    DBHOSTNAME="<database-server-name>"
```
Comment (jwendl) [inline PostGreKeyRotation/Deployment/scripts/pg-deploy-production.sh]: Let's move this to parameters.

**Review by jwendl — CHANGES_REQUESTED** [review]

File: PostGreKeyRotation/Deployment/pg-keyrotation-pipeline.yml (hunk 1 above)
Comment (ianisms) [reply PostGreKeyRotation/Deployment/pg-keyrotation-pipeline.yml]: Done

**Review by jwendl — APPROVED** [review]

**Review by jwendl — APPROVED** [review]

**Pull request merged**